src/test/common/expected-claim.js

```diff
@@ -0,0 +1,24 @@
+const { isNil } = require('ramda')
+
+const expectedClaim = {}
```

Comment:
Would we gain anything by making this module a class? It feels a bit odd mutating the `expectedClaim` constant. It might possibly be better to create an instance of `ExpectedClaim` and then call `expectedClaim.setProp(key, value)` (or something similar).